index.html

```diff
@@ -51,33 +47,33 @@ <h4>LET'S CONNECT</h4>
     <p>  <div class="item1">
```

Comment:
- [ ] `required`
- [ ] Amazing job so far 👏 
- [ ] However there is a few things which need slight adjustment like,
- [ ] fonts, and styling of the card, please make it looks as close as possible to the figma design.\
- [ ] please screenshot below for verification.
- [ ] ❗ `note this applies to all the cards you have made` ❗ 
`your template`
![image](https://user-images.githubusercontent.com/66230306/200038017-9c7c1690-3012-4395-bcd9-de99226f690f.png)

`figma template`
  
![image](https://user-images.githubusercontent.com/66230306/200038332-c6942344-46e3-4685-a831-8b63c8eed12a.png)